cmd/clairctl/clair/clair.go

```diff
@@ -0,0 +1,289 @@
+package clair
+
+import (
+	"strconv"
+	"strings"
+  "math"
```

Comment:
Have you ran any linter? Such as golint, gofmt -s, etc. Depending on your IDE, you may use https://github.com/alecthomas/gometalinter